File: Kotlin/src/main/kotlin/Java/Constructor.java
package Java;

public class Constructor {
    public static void main(String[] args) {
        Item item = new Item("연필", 1000);
        System.out.println("This name is " + item.name + ", " +
                "price is " + item.price);
    }
}
class Item{
    String name;
    int price;

    public Item(String name, int price){
        this.name=name;
        this.price=price;
    }
}

File: Kotlin/src/main/kotlin/Kotlin/클래스/Constructor.kt
package Kotlin.클래스

fun main() {
    var a = Constructor("박보영", 1990)
    var b = Constructor("전정국", 1997)
    var c = Constructor("장원영", 2004)

    var d = Constructor("이루다")
    var e = Constructor("차은우")
    var f = Constructor("류수정")

}
class Constructor (var name: String, var birthYear: Int){
    init {
        println("${this.birthYear}년생 ${this.name}님이 생성됨")
    }

    constructor(name:String) : this(name, 1997) {
        println("보조 생성자 생성됨")
    }
}